### PR TITLE
toString() for convenience and diagnostics purposes

### PR DESCRIPTION
**Pull request by ceefour**

### CODE REVIEW

File: src/main/java/fi/foyt/foursquare/api/entities/CompactVenue.java [hunk 1]
```diff
@@ -161,22 +163,44 @@ public HereNow getHereNow() {
...
-  private Stats stats;
-  private String url;
-  private Hours hours;
-  private Hours popular;
-  private Menu menu;
-  private Price price;
-  private Integer rating;
-  private SpecialGroup specials;
-  private HereNow hereNow;
+  protected String id;
```
Comment (clinejj) [inline src/main/java/fi/foyt/foursquare/api/entities/CompactVenue.java]: Why was the scope change done here?

File: src/main/java/fi/foyt/foursquare/api/entities/CompleteUser.java [hunk 2]
```diff
@@ -139,7 +139,6 @@ public Scores getScores() {
     return scores;
   }
 
-  private String type;
```
Comment (clinejj) [inline src/main/java/fi/foyt/foursquare/api/entities/CompleteUser.java]: why was this removed?

File: src/main/java/fi/foyt/foursquare/api/entities/List.java [hunk 3]
```diff
@@ -44,6 +44,34 @@
   private String createdAt;
   private String updatedAt;
   private String entities;
-  private Boolean isPublic; public void setPublic(Boolean isPublic) { this.isPublic = isPublic; }
+  private Boolean isPublic;
+  public void setPublic(Boolean isPublic) { this.isPublic = isPublic; }
```
Comment (clinejj) [inline src/main/java/fi/foyt/foursquare/api/entities/List.java]: if we're modifying this line i'd rather pull this function out into the rest of the class with appropriate formatting:
```
public void setPublic(Boolean isPublic) {
    this.isPublic = isPublic;
}
```

File: src/main/java/fi/foyt/foursquare/api/entities/CompactVenue.java (hunk 1 above)
Comment (ceefour) [inline src/main/java/fi/foyt/foursquare/api/entities/CompactVenue.java]: So they can be referred by subclass's toString

File: src/main/java/fi/foyt/foursquare/api/entities/CompleteUser.java (hunk 2 above)
Comment (ceefour) [inline src/main/java/fi/foyt/foursquare/api/entities/CompleteUser.java]: It hides the superclass's type field, same thing

File: src/main/java/fi/foyt/foursquare/api/entities/List.java (hunk 3 above)
Comment (ceefour) [inline src/main/java/fi/foyt/foursquare/api/entities/List.java]: Sure ! :)

Comment (ceefour) [inline src/main/java/fi/foyt/foursquare/api/entities/List.java]: @clinejj I've made your suggested fix

**Comment by clinejj** [comment]
@ceefour Thanks for these changes! My apologies for being so very late on merging them.
